MapboxNavigation/Error.swift

```diff
@@ -1,19 +1,14 @@
 import Foundation
-
-
-/**
- Custom Error code key used in returned in `voiceController(_:didInterrupt:with:)`.
- */
-public let SpokenInstructionErrorCodeKey = "SpokenInstructionErrorCodeKey"
+import MapboxCoreNavigation
 
 extension NSError {
     /**
      Creates a custom `Error` object.
      */
-    public convenience init(localizedFailureReason: String, detailedFailureReason: String, code: MapboxNavigationError = .defaultError) {
-        self.init(domain: MGLErrorDomain, code: code.rawValue, userInfo: [
+    public convenience init(localizedFailureReason: String, code: MapboxNavigationError = .defaultError) {
```

Comment:
This initializer is only useful to this SDK’s code, so make it private if possible.